File: latian/php/ListMhs.php
<?php
class ListMhs
{
    private $nama = '';
    private $nim = '';
    private $prodi = '';
    private $fakultas = '';

    public function __construct($nama = '', $nim = '', $prodi = '', $fakultas = '')
    {
        $this->nama = $nama;
        $this->nim = $nim;
        $this->prodi = $prodi;
        $this->fakultas = $fakultas;
    }

    public function setNama($nama)
    {
        $this->nama = $nama;
    }
    public function setNim($nim)
    {
        $this->nim = $nim;
    }
    public function setProdi($prodi)
    {
        $this->prodi = $prodi;
    }
    public function setFakultas($fakultas)
    {
        $this->fakultas = $fakultas;
    }

    public function getNama()
    {
        return $this->nama;
    }
    public function getNim()
    {
        return $this->nim;
    }
    public function getProdi()
    {
        return $this->prodi;
    }
    public function getFakultas()
    {
        return $this->fakultas;
    }
}

File: latian/python/ListMhs.py
class ListMhs:
    __nama = ""
    __nim = ""
    __prodi = ""
    __fakultas = ""

    def __init__(self, nama = "", nim = "",prodi = "",fakultas = ""):
        self.nama = nama
        self.nim = nim
        self.prodi = prodi
        self.fakultas = fakultas

    def get_nama(self):
        return self.nama
    def get_nim(self):
        return self.nim
    def get_prodi(self):
        return self.prodi
    def get_fakultas(self):
        return self.fakultas

    def set_nama(self, nama):
        self.__nama = nama
    def set_nim(self, nim):
        self.__nim = nim
    def set_prodi(self, prodi):
        self.__prodi = prodi
    def set_fakultas(self, fakultas):
        self.__fakultas = fakultas